src/test/java/org/sagebionetworks/bridge/services/StudyServiceMockTest.java

```diff
@@ -940,25 +941,23 @@ public void createStudyAndUser() throws SynapseException {
 
         // stub
         when(mockParticipantService.createParticipant(any(), any(), anyBoolean())).thenReturn(mockIdentifierHolder);
-        doNothing().when(mockSynapseClient).newAccountEmailValidation(any(), any());
 
         // execute
         service.createStudyAndUsers(mockStudyAndUsers);
 
         // verify
-        verify(mockParticipantService, times(2)).createParticipant(any(), any(), anyBoolean());
```

Comment:
Don't we still call createParticipant()?

Reply:
The next two lines verify the specific createParticipant calls, so this seems redundant.